src/features/commands.ts

```diff
@@ -638,47 +638,119 @@ Instead:
   },
   {
     words: ["!lock"],
-    help: "",
-    category: "Communication",
+    help: "used for locking channel",
+    category: "Moderation",
     handleMessage: async (msg) => {
       if (!msg.guild || !isStaff(msg.member)) {
-        return;
-      }
-
-      // permission overwrites can only be applied on Guild Text Channels
-      if (msg.channel.type === "GUILD_TEXT") {
-        const { channel: guildTextChannel } = msg;
-        await guildTextChannel.permissionOverwrites.create(
-          guildTextChannel.guild.roles.everyone,
-          {
-            SEND_MESSAGES: false,
-            CREATE_PUBLIC_THREADS: false,
-            CREATE_PRIVATE_THREADS: false,
-            SEND_MESSAGES_IN_THREADS: false,
-          },
-        );
-        guildTextChannel.send("This channel has been locked by a moderator");
+        await msg.react("💩")
+        await msg.channel.send({
+          embeds: [
+            {
+              title: "tsk tsk",
+              type: "rich",
+              description: `${msg.author.toString()} This command is reserved for staff, Dont try to do this kind of shit`,
+              color: EMBED_COLOR,
+            },
+          ],
+        })
```

Comment:
Can you remove this and the `!unlock` changes? I appreciate the reasoning, but I think it's fine to just ignore commands from users who can't use them. It's good to keep PRs focused on 1 change at a time

Reply:
Small change, so that's why i mixed with this

Reply:
This isn't the behavior we want from `!lock` and `!unlock`, so we can't merge the PR until this and other requested changes have been made